Resources/public/js/views/ez-languageswitcherview.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-languageswitcherview', function (Y) {
+    "use strict";
+    /**
+     * Provides the Language Switcher View class
+     *
+     * @module ez-languageswitcherview
+     */
+    Y.namespace('eZ');
+
+    var EXPANDED_LANGUAGE_LIST = 'is-expanded';
+
+    /**
+     * The raw content view
+     *
+     * @namespace eZ
+     * @class LanguageSwitcherView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.LanguageSwitcherView = Y.Base.create('languageSwitcherView', Y.eZ.TemplateBasedView, [Y.eZ.AccordionElement], {
+        events: {
+            '.ez-selected-language': {
+                'tap': '_expandList'
+            },
+            '.ez-content-translation a': {
+                'tap': '_switchLanguage'
+            }
+        },
+
+        initializer: function () {
+        },
+
+        /**
+         * Renders the language switcher view
+         *
+         * @method render
+         * @return {eZ.LanguageSwitcherView} the view itself
+         */
+        render: function () {
+            var container = this.get('container');
+
+            container.setHTML(this.template({
+                content: this.get('content').toJSON(),
+                //todo: get current translation and pass it here
+                currentTranslation: {'flagCode': 'gb', 'language': 'eng-GB', 'languageCode': 'eng-GB'},
+                translations: this._getOtherTranslationsList()
+            }));
+
+            return this;
+        },
+
+        /**
+         * Returns array with translations other than already being viewed
+         *
+         * @method _getOtherTranslationsList
+         * @private
+         * @return {Array} other translations
+         */
+        _getOtherTranslationsList: function () {
+            //todo: get list of existing translations for current content (miss current translation?)
```

Comment:
actually, the languages list is available in the version not in the content. When the content is loaded, the version info are also available (under Content.CurrentVersion.Version in the REST Response), so you should probably change the Content model to store that part somewhere and add a method to the Content model to get the corresponding Version.